src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs

```diff
@@ -107,6 +109,8 @@ public bool WaitForRequestHandlerConnection(int clientConnectionTimeout)
         /// <inheritdoc/>
         public async Task<int> InitializeCommunicationAsync(int clientConnectionTimeout)
         {
+            EqtTrace.Info("VsTestConsoleRequestSender.InitializeCommunicationAsync: Initializing Communication with vstest.console.exe.");
```

Comment:
Message, Say it started, and probably add ended at the end of this api

Reply:
Done